cmd/api/src/api/v2/auth/auth.go

```diff
@@ -191,7 +191,7 @@ func (s ManagementResource) DeleteSAMLProvider(response http.ResponseWriter, req
 		api.HandleDatabaseError(request, response, err)
 	} else if err := s.disassociateUsersFromSAMLProvider(request, providerUsers); err != nil {
 		api.HandleDatabaseError(request, response, err)
```

Comment:
This isn't needed anymore as well right?

Reply:
Just tested locally, and we indeed do not 😄